mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -472,15 +478,24 @@
     <!-- Notifications -->
     <!-- Text shown in snackbar when user deletes a collection -->
     <string name="snackbar_collection_deleted">Collection deleted</string>
+    <!-- Text shown in snackbar when user deletes a collection -->
```

Comment:
Shouldn't the comment be "Text shown in snackbar when user *renames* a collection"? (vs "delete")